WooCommerce/Classes/ViewRelated/Products/Cells/ProductsTabProductTableViewCell.swift

```diff
@@ -155,7 +162,7 @@ private extension ProductsTabProductTableViewCell {
 
     func configureNameLabel() {
         nameLabel.applyBodyStyle()
-        nameLabel.numberOfLines = 2
+        nameLabel.numberOfLines = 3
```

Comment:
Maybe this should be just `0`, what do you think?

Reply:
If this is 3, then `nameLabel` will be clipped after 3 lines, Since it's a name we may not need more than 3 lines. If it's okay to display more than 3 lines we can keep it 0.

Reply:
Since a variation name depends on how many attributes it has, I think it would be fairly normal for the name to spawn over 3 lines. So I think 0 makes sense here.

Reply:
Sure, I will change it to 0.